tutorials/euclid_access/4_Euclid_intro_PHZ_catalog.md

```diff
@@ -190,53 +189,47 @@ im_cutout= 5 * u.arcmin
 ra_cutout = 267.8
 dec_cutout =  66
 
-coords_cutout = SkyCoord(ra_cutout, dec_cutout, unit=(u.deg, u.deg), frame='icrs')
-##########################################################################
-im_cutout_deg=im_cutout.to(u.deg).value
-
-## Create ra and dec bounds for the box
-ra0=ra_cutout-im_cutout_deg/2
-ra1=ra_cutout+im_cutout_deg/2
-
-dec0=dec_cutout-im_cutout_deg/2
-dec1=dec_cutout+im_cutout_deg/2
+coords_cutout = SkyCoord(ra_cutout, dec_cutout, unit='deg', frame='icrs')
+size_cutout = im_cutout.to(u.deg).value
 ```
 
 ```{code-cell} ipython3
-adql = f"SELECT DISTINCT mer.object_id,mer.ra, mer.dec, phz.flux_vis_unif, phz.flux_y_unif, \
-phz.flux_j_unif, phz.flux_h_unif, phz.phz_classification, phz.phz_median, phz.phz_90_int1, phz.phz_90_int2 \
-FROM {table_mer} AS mer \
-JOIN {table_phz} as phz \
-ON mer.object_id = phz.object_id \
-WHERE 1 = CONTAINS(POINT('ICRS', mer.ra, mer.dec), CIRCLE('ICRS', {ra_cutout}, {dec_cutout}, {im_cutout_deg/2})) \
-AND phz.flux_vis_unif> 0 \
-AND  phz.flux_y_unif > 0 \
-AND phz.flux_j_unif > 0 \
-AND phz.flux_h_unif > 0 \
-AND phz.phz_classification = 2 \
-AND ((phz.phz_90_int2 - phz.phz_90_int1) / (1 + phz.phz_median)) < 0.20 \
-AND phz.phz_median BETWEEN 1.4 AND 1.6 \
-"
-adql
+adql = ("SELECT DISTINCT mer.object_id, mer.ra, mer.dec, "
+        "phz.flux_vis_unif, phz.flux_y_unif, phz.flux_j_unif, phz.flux_h_unif, "
+        "phz.phz_classification, phz.phz_median, phz.phz_90_int1, phz.phz_90_int2 "
+        f"FROM {table_mer} AS mer "
+        f"JOIN {table_phz} as phz "
+        "ON mer.object_id = phz.object_id "
+        "WHERE 1 = CONTAINS(POINT('ICRS', mer.ra, mer.dec), "
+                            f"BOX('ICRS', {ra_cutout}, {dec_cutout}, {size_cutout/np.cos(coords_cutout.dec)}, {size_cutout})) "
```

Comment:
note that I switched to do a BOX search rather than a cone search to be consistent with the existing narrative and to have full coverage for our cutout image (e.g. there were a couple of galaxies missed in the corners of the image with the previous usage).